save-demo/src/main/kotlin/com/saveourtool/save/demo/storage/ToolStorage.kt

```diff
@@ -14,9 +17,29 @@ private const val TOOLS_PATH = "tools"
  */
 @Component
 class ToolStorage(
-    private val configProperties: ConfigProperties,
+    configProperties: ConfigProperties,
 ) : AbstractFileBasedStorage<ToolKey>(Path.of(configProperties.fileStorage.location) / TOOLS_PATH) {
-    private val rootPath = Path.of(configProperties.fileStorage.location) / TOOLS_PATH
+    /**
+     * Todo: Implement DownloadToolService
+     */
+    init {
```

Comment:
Can be done as `@PostConstruct` on a class method. Then Spring will take care of initialization during proper lifecycle phase

Reply:
Wow, thanks, looks much better

Reply:
Done